apis/v1beta1/gateway_types.go

```diff
@@ -70,7 +70,12 @@ type GatewaySpec struct {
 	// At least one Listener MUST be specified.
 	//
 	// Each listener in a Gateway must have a unique combination of Hostname,
-	// Port, and Protocol.
+	// Port, and Protocol. Combinations that are considered Core are:
```

Comment:
```suggestion
	// Port, and Protocol. Below combinations are considered Core and must be supported:
```

Reply:
Good catch, this is better. One tiny additional suggestion on top of this - we usually capitalize "MUST" in the API Spec.